src/SerializerInterface.php

```diff
@@ -11,6 +11,14 @@
  */
 interface SerializerInterface
 {
+    public const FORMAT_JSON = 'json';
+    public const FORMAT_XML = 'xml';
+
+    public const FORMATS_SUPPORTED = [
```

Comment:
not sure if this is a good idea. 
Having this constants in the serializer interface does not allow to add formats (visitors) without changing this interface.

This makes useless the idea of being able to add your own visitors.